pkg/gadgets/process-collector/tracer/bpf/process-collector.c

```diff
@@ -12,11 +12,6 @@
 #include <vmlinux/vmlinux.h>
 
 #include <bpf/bpf_helpers.h>
-/* libbpf v0.4.0 introduced BPF_SEQ_PRINTF in bpf_tracing.h.
- * In future versions, it will be in bpf_helpers.h.
- */
-#include <bpf/bpf_tracing.h>
```

Comment:
I tried `go generate ./...` on this branch, and I get:
```
/home/alban/go/src/github.com/kinvolk/inspektor-gadget/pkg/gadgets/process-collector/tracer/bpf/process-collector.c:37:2: error: implicit declaration of function 'BPF_SEQ_PRINTF' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        BPF_SEQ_PRINTF(seq, "%d %d %s %llu\n", task->tgid, task->pid, task->comm, mntns_id);
        ^
1 error generated.
Error: clang: exit status 1
```

It works when I reintroduce `bpf/bpf_tracing.h`.

```
$ rpm -qa|grep  libbpf
libbpf-0.4.0-1.fc34.x86_64
libbpf-devel-0.4.0-1.fc34.x86_64
```

Do we want to keep supporting libbpf 0.4.0?

Ditto in pkg/gadgets/socket-collector/tracer/bpf/tcp4-collector.c

Reply:
> Do we want to keep supporting libbpf 0.4.0?

libbpf v0.4.0 was released more than a year ago (https://github.com/libbpf/libbpf/releases/tag/v0.4.0) and the latest release now is v0.8.0 (https://github.com/libbpf/libbpf/releases/tag/v0.8.0). IMO we could require a newer version. 

Also, libbpf is only needed for the bpf headers, there is https://github.com/cilium/ebpf/issues/449 that could even remove this dependency for us.

Reply:
I use Fedora 34 and it only has libbpf 0.4.0. Since this PR removes the ebpf build in a container, it means I have to update libbpf on my laptop. I guess it is time to update it :)